installation/sql/postgresql/base.sql

```diff
@@ -1149,7 +1155,8 @@ CREATE INDEX "#__workflow_stages_idx_checked_out" ON "#__workflow_stages" ("chec
 --
 
 INSERT INTO "#__workflow_stages" ("id", "asset_id", "ordering", "workflow_id", "published", "title", "description", "default") VALUES
-(1, 57, 1, 1, 1, 'COM_WORKFLOW_BASIC_STAGE', '', 1);
+(1, 57, 1, 1, 1, 'COM_WORKFLOW_BASIC_STAGE', '', 1),
+(2, 90, 1, 2, 1, 'COM_WORKFLOW_BASIC_WORKFLOW_MODULES', '', 1);
 
 SELECT setval('#__workflow_stages_id_seq', 2, false);
```

Comment:
```suggestion
SELECT setval('#__workflow_stages_id_seq', 3, false);
```
That one I almost had forgotten.